### PR TITLE
nodemon: make --inspect a generic example

### PR DESCRIPTION
**Pull request by schwarmco**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

If you give `--inspect` as an argument to nodemon, it passes it through to nodejs, which then listens for a Debugger. It is not meant as "inspect the given file for changes".

### CODE REVIEW

**Comment by navarroaxel** [comment]
Hi @schwarmco, welcome and thanks for your submission. I think you can add a new example specific for `--inspect`. What do you think?

**Comment by schwarmco** [comment]
Hi @navarroaxel, thank you for your feedback on that. As `--inspect` is only an example of what argument could be passed down to node here, what do you think of the example i've added?

File: pages/common/nodemon.md [hunk 1]
```diff
@@ -19,6 +19,10 @@
 
 `nodemon {{path/to/file.js}} {{arguments}}`
 
+- Pass arguments to node itself if they're not nodemon arguments already (e.g. --inspect):
```
Comment (marchersimon) [inline pages/common/nodemon.md:22]:
```suggestion
- Pass arguments to node itself if they're not nodemon arguments already (e.g. `--inspect`):
```

**Review by marchersimon — APPROVED** [review]
Looks good, I've just left a small suggestion

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**